packages/cashier/src/components/cashier-locked/__tests__/cashier-locked.spec.tsx

```diff
@@ -1,51 +1,42 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import CashierLocked from '../cashier-locked';
-import { useDepositLocked } from '@deriv/hooks';
-import { TRootStore } from 'Types';
+import { useCashierLocked, useDepositLocked } from '@deriv/hooks';
+import { mockStore } from '@deriv/stores';
+import { TRootStore } from '../../../types';
```

Comment:
@rostislav-deriv Please don't use `TRootStore` as it's deprecated, Actually you don't even need the type so you can remove it in all the test files 🙏🏻

Reply:
Removed it